swan/README.md

```diff
@@ -0,0 +1,20 @@
+# Swan
+
+This folder contains  Notecard Outboard Firmware Update examples specifically for the [Swan](https://blues.io/products/swan/) MCU.
+
+Three different applications are presented here implemented across several runtime environments:
+
+* Circuit Python
+* Arduino
+* Zephyr
```

Comment:
It appears the binaries (and corresponding checksums) related to Zephyr were deleted.

Is this bullet still valid, or am I missing the binaries?

Reply:
It's still valid - the zephyr code is still there for the `button-press` app